### DISCUSSION TITLE
Authentication Built in database

### DISCUSSION THREAD

**Discussion by sapl-santhosh** [emqx/emqx#13870] [General]
How to take backup of the built in database for password - authentication. 

Use case. when something goes wrong in the underlying OS Ubuntu or when needed to upgrade the OS. 
My server is running on Ubuntu 22.04

**Comment by ieQu1** [comment]
Backup/restore feature is designed for this:
https://docs.emqx.com/en/emqx/latest/operations/backup-restore.html#backup-and-restore

**Comment by sapl-santhosh** [comment]
But it seems it is available only in the enterprise version?

Thanks & Regards

S.Santhosh Kumar

**Reply by ieQu1** [reply]
No, `emqx_ctl data export/import` CLI commands are available in all editions.

**Comment by sapl-santhosh** [comment]
but i cannot see Backup and Restore option in System section in Emqx
dashboard

Thanks & Regards

S.Santhosh Kumar

**Reply by id** [reply]
@sapl-santhosh this functionality is available via CLI and API in the Community edition. Enterprise edition also supports this in the UI (Dashboard).
